### DISCUSSION TITLE
Can't verify my academic affiliation

### DISCUSSION THREAD

**Discussion by k-kuroguro** [community/community#50954] [GitHub Education] [answered]
I received the following message:

- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
- You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

However, I attend the university in person and my current location is also about an hour away. Could it be that my device's location information is incorrect? My university has campuses in Japan and Chicago, so I suspect the system may be confusing this.

Thank you in advance.

**Comment by samus-aran** [comment, marked as answer]
Hi @k-kuroguro 👋 

Thank you for bringing this to our attention. We've updated the details on your selected school and we would like to ask you to re-apply! Let us know how you get on 👍

**Reply by k-kuroguro** [reply]
Hi @samus-aran

Thank you for your response. I was able to participate in GitHub Education without any issues!

**Reply by samus-aran** [reply]
That's awesome @k-kuroguro  ! 🎉 Thanks so much for coming back and sharing the update. This will help other community members seeking similar answers. Start enjoying GitHub Global Campus 🌟:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋

**Comment by Aarjav0210** [comment]
Hi there! @samus-aran

I received the same message; however, I am currently studying in my year abroad. This is indicated on the form of proof I upload along with my up-to-date term-time address (which is not in the same country). I suspect my current location is throwing off the system, but I have provided all the information, so I wonder if this could be resolved soon.

Thank you in advance.

**Reply by samus-aran** [reply]
Hi @Aarjav0210 👋 

This discussion is now closed. We would recommend you open up a new discussion with your situation. @k-kuroguro issue was resolved because the details on their selected school were incorrect. That doesn't sound like the same situation you are in. This is why we'll ask you to open a separate discussion to converse further 👍

**Comment by Akash1491625** [comment]
How to reapply to the github student pack,beacuse i get failed 2 times before.

**Reply by samus-aran** [reply]
Hi @Akash1491625  👋

This discussion is now closed. We would recommend you open up a new discussion with your situation. The original poster's issue was resolved because the details on their selected school were incorrect. That doesn't sound like the same situation you are in. This is why we'll ask you to open a separate discussion to converse further 👍
